### ISSUE TITLE
Immense slowdown if Spigot gets disconnected and tries to retrieve something from the bungee (?)

### ISSUE THREAD

**Issue by Zulfen** [Skungee/Skungee-2.0.0#19]
## Description

I have noticed that (at least for me), if Skungee can't connect to the Bungee, it will drastically reduce my TPS down and makes the server unplayable. I am guessing this is due to it trying to reconnect? This isn't a hardware issue either - server runs at a solid 20 TPS if it connects properly.

## Screenshots or Errors

No errors related to Skungee, but here you can see the server skipping ticks here:

```
>.... [19:18:30 WARN]: Can't keep up! Is the server overloaded? Running 5053ms or 101 ticks behind
>.... [19:19:17 WARN]: Can't keep up! Is the server overloaded? Running 5042ms or 100 ticks behind
>.... [19:20:01 WARN]: Can't keep up! Is the server overloaded? Running 5010ms or 100 ticks behind
```

## Steps to Reproduce the Problem

1. Setup a normal Skungee environment (bungee and spigot servers)
2. Shutdown the Bungeecord server.

 ## Specifications

Skungee Version: Skungee v2.0.0-ALPHA-7 (says 6 in console still)
Proxy Platform and Version: Waterfall-366 and Paper-373 (1.15.2)
System/Panel (if applicable): Linux (not sure what distro), Java 8

**Comment by Zulfen** [comment]
Oh yeah - forgot to send the code I am using! Whoops!:

```
every 3 seconds:
   set {_players::*} to all proxied players
```

(using this for my sidebar)

**Comment by ardaok** [comment]
yeah this is urgent it blows my server

**Comment by TheLimeGlass** [comment]
Fixed in 2.0.0-ALPHA-9
